File: BaekJoon/BasicMath2/3009.py
x1, y1 = map(int, input().split())
x2, y2 = map(int, input().split())
x3, y3 = map(int, input().split())

x = list(filter(lambda x: [x1,x2,x3].count(x) == 1,[x1,x2,x3]))[0]
y = list(filter(lambda y: [y1,y2,y3].count(y) == 1,[y1,y2,y3]))[0]

print(str(x)+" "+str(y))

File: BaekJoonSwift/BaekJoonSwift/Basic Math2 (1)/3009.swift
import Foundation

var x = [Int]()
var y = [Int]()

for _ in 0..<3 {
    let input = readLine()!.split(separator: " ").map { Int($0)! }
    x.append(input[0])
    y.append(input[1])
}

x = x.filter { x.lastIndex(of: $0) == x.firstIndex(of: $0) }
y = y.filter { y.lastIndex(of: $0) == y.firstIndex(of: $0) }

print(x[0], y[0])
